parm/config/gfs/config.sfcanl

```diff
@@ -12,5 +12,11 @@ echo "BEGIN: config.sfcanl"
 if [[ "${DO_JEDIATMVAR}" == "YES" ]]; then
    export DONST="NO"
 fi
+if [[ "${GSI_SOILANAL}" = "YES" ]]; then
+    export DO_LNDINC=".true."
+    export DO_SOI_INC=".true."
+    export GCYCLE_INTERP_LNDINC=".false."
+    export LSOIL_INCR=2
```

Comment:
This is a nitpicky comment - and is a case in the global-workflow in general - variables do not need to be 3 characters long e.g. `DO_SOI_INC -> DO_SOIL_INC` or `DO_SOILINC` to be (in)consistent with `DO_LNDINC` or something.  Even `INC` is sometimes `INCR`.

Reply:
I'm preparing to fix this, with a second PR to UFS_UTILS, resolving issue [1021](https://github.com/ufs-community/UFS_UTILS/issues/1021)